### DISCUSSION TITLE
How to get coefficients from a formula?

### DISCUSSION THREAD

**Discussion by lalili** [axkr/symja_android_library#299] [General]
Hello 
If I have a formula, can I get a coefficient for every one of the variables? 
For example, if I set F1 = 0. F2 = 1, I will get the coefficient of F1 (=10)
Is there a better way to get coefficients from every one of the variables (F1, F2, F3)?
```
ExprEvaluator util = new ExprEvaluator();
util.defineVariable("F1", 0);
util.defineVariable("F2", 1);
util.defineVariable("F3", 0);
IExpr exp2 = util.evaluate("F1+10*F2+15*F3").evalDouble();//10
```

thank you

**Comment by axkr** [comment]
You can use some of these functions for polynomials:
- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/Coefficient.md
- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/CoefficientList.md

- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/Exponent.md
- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/MonomialList.md
